### PR TITLE
Update using-gulp.md

### PR DESCRIPTION
**Pull request by songgun**
Line 103: ASP.NET Web Application project - > ASP.NET Web Core Application project
Line 246: `series:second` - > `series`
Line 269~278: Sample code replacement - The context and description are based on CSS files, and the example code is about .js files.
Line 293, 303: Hosting:Environment -> ASPNETCORE_ENVIRONMENT

### CODE REVIEW

File: aspnetcore/client-side/using-gulp.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ The following table provides an explanation of the tasks specified in the code a
 
 ## Running default tasks
 
-If you haven't already created a new Web app, create a new ASP.NET Web Application project in Visual Studio.
+If you haven't already created a new Web app, create a new ASP.NET Web Core Application project in Visual Studio.
```
[the remark this answers is not in the document]
Comment (songgun) [reply aspnetcore/client-side/using-gulp.md]: It is a good suggestion. To agree.

File: aspnetcore/client-side/using-gulp.md [hunk 2]
```diff
@@ -243,7 +243,7 @@ When you run multiple tasks, the tasks run concurrently by default. However, if
 	gulp.task('default', gulp.series('series'));
     ```
  
-    You now have three tasks: `series:first`, `series:second`, and `series`. The `series:second` task includes a second parameter which specifies an array of tasks to be run and completed before the `series:second` task will run. As specified in the code above, only the `series:first` task must be completed before the `series:second` task will run.
+    You now have three tasks: `series:first`, `series:second`, and `series`. The `series` task includes a second parameter which specifies an array of tasks to be run and completed before the `series:second` task will run. As specified in the code above, only the `series:first` task must be completed before the `series` task will run.
```
[the remark this answers is not in the document]
Comment (songgun) [reply aspnetcore/client-side/using-gulp.md]: It is a good suggestion. To agree.

File: aspnetcore/client-side/using-gulp.md [hunk 3]
```diff
@@ -300,7 +290,7 @@ To switch between compiling for different environments, modify the **ASPNETCORE_
 
 3.  Click the **Debug** tab.
 
-4.  Set the value of the **Hosting:Environment** environment variable to `Production`.
+4.  Set the value of the **ASPNETCORE_ENVIRONMENT** environment variable to `Production`.
```
[the remark this answers is not in the document]
Comment (songgun) [reply aspnetcore/client-side/using-gulp.md]: It is a good suggestion. To agree.

File: aspnetcore/client-side/using-gulp.md [hunk 4]
```diff
@@ -310,7 +300,7 @@ To switch between compiling for different environments, modify the **ASPNETCORE_
 
 7.  Close the browser to stop the Web app.
 
-8.  In Visual Studio, return to the property sheet for the Web app and change the **Hosting:Environment** environment variable back to `Development`.
+8.  In Visual Studio, return to the property sheet for the Web app and change the **ASPNETCORE_ENVIRONMENT** environment variable back to `Development`.
```
[the remark this answers is not in the document]
Comment (songgun) [reply aspnetcore/client-side/using-gulp.md]: It is a good suggestion. To agree.

**Comment by scottaddie** [comment]
@songgun Please click the **Commit suggestion** button below each suggested change to apply it. I'll take another look once that's done.

**Pull request closed without merging**